pkg/config/apiserver.go

```diff
@@ -114,6 +114,52 @@ func (a *API) register(c *restful.Container) {
 		Reads(&pb.ServiceConfig{}).
 		Writes(APIResponse{}))
 
+	// Adapters
+	ws.Route(ws.
```

Comment:
yes, we will add query params at that time, but that goes hand in hand with `PATCH` operations.

This method is used when the operator is editing entire files, (like the config is checked into source control), and you want to push it into Mixer.